strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/artifact/generator/MavenArtifactGenerator.java

```diff
@@ -316,4 +318,11 @@ public void setBasedir(String basedir)
         this.basedir = basedir;
     }
 
+    @Override
+    public Path generateArtifact(URI uri,
+                                 int size)
+            throws IOException
+    {
+        return null;
```

Comment:
I think that these methods need to be implemented.

Reply:
Thanks for letting me know. Actually issue [#1145](https://github.com/strongbox/strongbox/issues/1145) description says -  We need to make all this classes implement the` org.carlspring.strongbox.artifact.generator.ArtifactGenerator` interface.
Now after doing the above I did think about whether I need to do the implementation. For Example: overridden method `generateArtifact` takes 2 args `URI uri, int size` but I am not sure how to use them.